src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -120,6 +120,14 @@ public class HubEndPoint<THub> : IInvocationBinder where THub : Hub
                             ? (IDataEncoder)Base64Encoder
                             : PassThroughEncoder;
 
+                        var transferMode = connection.Features.Get<ITransferModeFeature>() ??
+                            throw new InvalidOperationException("Unable to read transfer mode.");
+
+                        transferMode.TransferMode =
```

Comment:
https://github.com/aspnet/SignalR/blob/29f9c54b864d3f86522b15651acf3b6adc57c8be/src/Microsoft.AspNetCore.Sockets.Http/Internal/Transports/WebSocketsTransport.cs#L181-L183
- I must have lost it in one of rebases/merges.